flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBWriteBatchWrapperTest.java

```diff
@@ -65,4 +69,52 @@ public void basicTest() throws Exception {
 			}
 		}
 	}
+
+	/**
+	 * Tests that {@link RocksDBWriteBatchWrapper} flushes after the memory consumed exceeds the preconfigured value.
+	 */
+	@Test
+	public void testWriteBatchWrapperFlushAfterMemorySizeExceed() throws Exception {
+		try (RocksDB db = RocksDB.open(folder.newFolder().getAbsolutePath());
+			WriteOptions options = new WriteOptions().setDisableWAL(true);
+			ColumnFamilyHandle handle = db.createColumnFamily(new ColumnFamilyDescriptor("test".getBytes()));
+			RocksDBWriteBatchWrapper writeBatchWrapper = new RocksDBWriteBatchWrapper(db, options, 200, 50)) {
+			// sequence (8 bytes) + count (4 bytes)
+			// more information please ref to write_batch.cc in RocksDB
+			assertEquals(12, writeBatchWrapper.getDataSize());
```

Comment:
This assertion might fail if the rocksdb implementation changes and is unnecessary. We could simply record an `initialSize` for `writeBatchWrapper.getDataSize()` and check against it after flush is triggered.

Reply:
done